packages/react-next/src/components/OverflowSet/OverflowSet.base.tsx

```diff
@@ -1,257 +1,225 @@
-      items &&
-        items.forEach(overflowItem => {
-          const keytip = (overflowItem as IOverflowSetItemProps).keytipProps;
-          if (keytip) {
-            // Create persisted keytip
-            const persistedKeytip: IKeytipProps = {
-              content: keytip.content,
-              keySequences: keytip.keySequences,
-              disabled: keytip.disabled || !!(overflowItem.disabled || overflowItem.isDisabled),
-              hasDynamicChildren: keytip.hasDynamicChildren,
-              hasMenu: keytip.hasMenu,
-            };
-
-            if (keytip.hasDynamicChildren || this._getSubMenuForItem(overflowItem)) {
-              // If the keytip has a submenu or children nodes, change onExecute to persistedKeytipExecute
-              persistedKeytip.onExecute = this._keytipManager.menuExecute.bind(
-                this._keytipManager,
-                overflowKeytipSequences,
-                overflowItem.keytipProps && overflowItem.keytipProps.keySequences,
-              );
-            } else {
-              // If the keytip doesn't have a submenu, just execute the original function
-              persistedKeytip.onExecute = keytip.onExecute;
-            }
-
-            // Add this persisted keytip to our internal list, use a temporary uniqueID (its content)
-            // uniqueID will get updated on register
-            this._persistedKeytips[persistedKeytip.content] = persistedKeytip;
-
-            // Add the overflow sequence to this item
-            const newOverflowItem = {
-              ...overflowItem,
-              keytipProps: {
-                ...keytip,
-                overflowSetSequence: overflowKeytipSequences,
-              },
-            };
-            newOverflowItems.push(newOverflowItem);
+      itemsProp.forEach(overflowItem => {
+        const keytip = (overflowItem as IOverflowSetItemProps).keytipProps;
+        if (keytip) {
+          // Create persisted keytip
+          const persistedKeytip: IKeytipProps = {
+            content: keytip.content,
+            keySequences: keytip.keySequences,
+            disabled: keytip.disabled || !!(overflowItem.disabled || overflowItem.isDisabled),
+            hasDynamicChildren: keytip.hasDynamicChildren,
+            hasMenu: keytip.hasMenu,
+          };
+          if (keytip.hasDynamicChildren || getSubMenuForItem(overflowItem)) {
+            // If the keytip has a submenu or children nodes, change onExecute to persistedKeytipExecute
+            persistedKeytip.onExecute = keytipManager.menuExecute.bind(
+              keytipManager,
+              overflowKeytipSequences,
+              overflowItem.keytipProps.keySequences,
+            );
           } else {
-            // Nothing to change, add overflowItem to list
-            newOverflowItems.push(overflowItem);
+            // If the keytip doesn't have a submenu, just execute the original function
+            persistedKeytip.onExecute = keytip.onExecute;
           }
```

Comment:
Small thing but please bring back the spaces between lines; they do help with readability.